### PR TITLE
Remove GH prefix

### PR DESCRIPTION
**Pull request by flurdy**
Removing the DEFRA prefix for the templates in frontend as superfluous.

### CODE REVIEW

**Review by bartosh1122 — APPROVED** [review]

File: src/server/services/transformers/service-status.test.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ describe('#serviceStatus', () => {
           pullRequest: {
             url: {
               href: `https://github.com/${githubOrg}/cdp-nginx-upstreams/pull/123`,
-              text: `${githubOrg}/cdp-nginx-upstreams/pull/123`
+              text: 'DEFRA/cdp-nginx-upstreams/pull/123'
```
Comment (chotai) [inline src/server/services/transformers/service-status.test.js:65]: `DEFRA` should this be in the test?

File: src/server/services/transformers/service-to-entity-data-list.test.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ describe('#serviceToEntityDataList', () => {
             kind: 'link',
             newWindow: true,
             url: `https://github.com/${githubOrg}/cdp-deployables`,
-            value: `${githubOrg}/cdp-deployables`
+            value: 'DEFRA/cdp-deployables'
```
Comment (chotai) [inline src/server/services/transformers/service-to-entity-data-list.test.js:20]: should we remove the `DEFRA` here too?

Comment (flurdy) [reply src/server/services/transformers/service-to-entity-data-list.test.js:20]: No, well, ideally, but it was the one case were the repo does not match the service name.

Comment (flurdy) [reply src/server/services/transformers/service-to-entity-data-list.test.js:20]: This whole test is an edge case that should not be possible to reach any more
